lib/Core/Executor.cpp

```diff
@@ -276,38 +250,115 @@ namespace {
 		    clEnumValN(Executor::User, "User", "Wrong klee_* functions invocation"),
 		    clEnumValN(Executor::Unhandled, "Unhandled", "Unhandled instruction hit")
 		    KLEE_LLVM_CL_VAL_END),
-		  cl::ZeroOrMore);
+		  cl::ZeroOrMore,
+                  cl::cat(TerminationCat));
 
   cl::opt<unsigned long long>
   StopAfterNInstructions("stop-after-n-instructions",
                          cl::desc("Stop execution after specified number of instructions (default=0 (off))"),
-                         cl::init(0));
+                         cl::init(0),
```

Comment:
Do we just groupings? For instance here I'd vote for renaming it to `max-instructions`.

Reply:
Good suggestion.